### ISSUE TITLE
Cannot load components ReferenceError: eventEmit is not defined (SSR)

### ISSUE THREAD

**Issue by PeterKnight** [surmon-china/vue-quill-editor#31]
I'm trying to get the editor to work in an electron app and followed along with the example in /nuxt-ssr-example, but I'm getting the error:

```
Cannot load components ReferenceError: eventEmit is not defined
```

for every interaction.

I've created a repo using electronuxt which is just a basic starter app for nuxt. 

#### Vue.js version and component version
I'm using vue-quill-editor 2.0.3 and nuxt 0.10.5

#### Reproduction Link
https://github.com/PeterKnight/quilltest

#### Steps to reproduce
Install the project with npm, load it with npm run dev, you should see the editor pop up ok but any interaction with the editor results in errors showing in the console.

#### What is Expected?		
For editor events to work properly

#### What is actually happening?
eventEmit not defined errors

**Comment by surmon-china** [comment]
Thanks for your feedback, I will soon solve this problem and should be in tomorrow this question will be solved!

**Comment by surmon-china** [comment]
I fixed this in v2.0.4, just update your dependency and it should work :) !

**Comment by PeterKnight** [comment]
YAY! It works now *dance*
